.gitmodules

```
@@ -47,3 +47,7 @@
 	path = src/tools/rustc-perf
 	url = https://github.com/rust-lang/rustc-perf.git
 	shallow = true
+[submodule "src/tools/enzyme"]
+	path = src/tools/enzyme
+	url = git@github.com:EnzymeAD/Enzyme.git
```

I think you should use the the `https` url instead of the `git` url.
```suggestion
	url = https://github.com/EnzymeAD/Enzyme.git
```

---

Yep, thanks you two!